CommunityToolShedMvc/CommunityToolShedMvc/ViewModels/AddCommunityViewModel.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace CommunityToolShedMvc.ViewModels
+{
+    public class AddCommunityViewModel
+    {
+        public int Id
+        {
+            get;set;
+        }
+        public string Name
```

Comment:
Any `string` properties that are required to have a value should be decorated with a `Required` data annotation attribute.